_docs/policies/tests.md

```diff
@@ -0,0 +1,47 @@
+# `status-go` Test Policy
+
+- [Creating Tests](#creating-tests)
+- [Flaky Tests](#flaky-tests)
+
+## Creating Tests
+
+- All new functionality MUST be introduced with tests that:
+  - Prove that the functionality performs as described
+  - Can be falsified
+  - Are resistant to fuzzing
+- All new tests MUST BE validated via 1000 local tests. Ensuring that the test runs and passes consistently every time gives confidence that the test is not flaky.
+    - add the `-count` flag to the test command eg: `-count 1000`
+    - OR wrap your test in a for loop
+        ```go
+        func TestTheThing(t *testing.T) {
+            for i := 1; i < 1000; i++ {
+                fmt.Println("Test Run", i)
+                // your test body goes in here
+            }
+        }
```

Comment:
Please don't :sweat_smile:. We'll end up with some tests wrapped, others not. This should definitely be done either by using the `count` flag or a custom script.

Reply:
😆 I should have added the 🙈 emoji and the made super clear that this is only for local usage. Nobody should try merging a test like that.

Reply:
Actually I do this time to time, when I want a breakpoint to hit on a flaky test

Reply:
I think the consensus here is to not recommend this approach. I'll remove this. Developers are free to dabble in the dark arts in the comfort of their own code editor.